hedgehog/src/Hedgehog/Internal/Report.hs

```diff
@@ -784,7 +784,7 @@ ppSummary summary =
             Nothing
         ]
 
-renderDoc :: MonadIO m => Maybe UseColor -> Doc Markup -> m String
+renderDoc :: MonadIO m => UseColor -> Doc Markup -> m String
 renderDoc mcolor doc = do
```

Comment:
This variable name is short for `maybe color`, if it's not a `Maybe` anymore we should rename it